src/Crawler.php

```diff
@@ -144,12 +144,19 @@ public function crawlSite( string $static_site_path ) : void {
             } elseif ( ! is_null( $crawled_contents ) ) {
                 $page_hash = md5( $crawled_contents );
             } else {
+                // TODO: Hashing the status code will generate many collisions
+                // - what are we trying to accomplish here?
```

Comment:
The hash is compared to the previous hash for the same URL to see if the page has changed at all. It won't collide because the hashed_url must also match.